src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -122,6 +41,13 @@ internal class SemanticTokensHelpers
                 [ClassificationTypeNames.StringLiteral] = LSP.SemanticTokenTypes.String,
             };
 
+        public static readonly ImmutableArray<string> RoslynCustomTokenTypes = ClassificationTypeNames.AllTypeNames
```

Comment:
I probably missed the context, is there a specific reason why this is more preferable than the prior (where we re-specified everything)? Asking only because I could also see a world where the "verification test" also goes as far as to ensure this `SemanticTokensHelpers` class is specifying everything in `ClassificationTypeNames`

Reply:
Good question! IMO there's a couple of reasons why I think this is more preferable than prior:
- `AllTypeNames` specifies every classification type and therefore can be easily re-used in non-LSP contexts. The previous `RoslynCustomTokenTypes` only specified a subset of classification types.
- Before, if we wanted to add a new mapping in `ClassificationTypeToSemanticTokenTypeMap`, it would be a two-step process: we'd have to add the mapping, and then delete the entry in `RoslynCustomTokenTypes`. With the updated code, we can just add a new entry to `ClassificationTypeToSemanticTokenTypeMap` and be done (besides needing to update the LSP client/Razor)

Essentially, I think it's just cleaner and allows for easier future maintainability 😄